### PR TITLE
doc: add a sql.DB integration section for pgx with PostgreSQL

### PR DESCRIPTION
**Pull request by roger2hk**
Add an extra section in doc for pgx with PostgreSQL according to the code provided by @a8m in https://github.com/facebookincubator/ent/issues/563.

### CODE REVIEW

File: doc/md/sql-integration.md [hunk 1]
```diff
@@ -103,3 +103,49 @@ func Open(dsn string) *ent.Client {
...
+	_ "github.com/jackc/pgx/v4/stdlib"
+)
+
+// Open new connection
+func Open(databaseUrl string) *ent.Client {
+	db, err := sql.Open("pgx", databaseUrl)
+	if err != nil {
+		log.Fatal(err)
+	}
+    defer db.Close()
```
Comment (a8m) [inline doc/md/sql-integration.md:130]: Can you fix this indentation? 
```suggestion
	defer db.Close()
```

File: doc/md/sql-integration.md [hunk 2]
```diff
@@ -103,3 +103,49 @@ func Open(dsn string) *ent.Client {
...
+
+	// Create an ent.Driver from `db`.
+    drv := entsql.OpenDB("postgres", db)
+    return ent.NewClient(ent.Driver(drv))
```
Comment (a8m) [inline doc/md/sql-integration.md:134]:
```suggestion
	drv := entsql.OpenDB("postgres", db)
	return ent.NewClient(ent.Driver(drv))
```

File: doc/md/sql-integration.md [hunk 3]
```diff
@@ -103,3 +103,49 @@ func Open(dsn string) *ent.Client {
...
+}
+
+func main() {
+    client := Open("postgresql://user:password@127.0.0.1/database")
```
Comment (a8m) [inline doc/md/sql-integration.md:138]:
```suggestion
	client := Open("postgresql://user:password@127.0.0.1/database")
```

File: doc/md/sql-integration.md [hunk 4]
```diff
@@ -103,3 +103,49 @@ func Open(dsn string) *ent.Client {
...
+
```
Comment (a8m) [inline doc/md/sql-integration.md:139]:
```suggestion
```

**Comment by roger2hk** [comment]
> Looks good! Thanks for adding this.
> 
> I added minor style comments.

Indentation fixed in https://github.com/facebookincubator/ent/pull/574/commits/70634e681b78026745f308457718485e0e575470. Thanks for the review.

**Pull request merged**